packages/nocodb/src/db/field-handler/index.ts

```
@@ -362,4 +415,121 @@ export class FieldHandler implements IFieldHandler {
+            //     row,
+            //     baseModel: baseModel,
+            //     column,
+            //     options: { ...params.options, fieldHandler: this },
+            //   })
+            // ).value;
+            // row[column.title] = value;
+            // row[column.id] = value;
+          }
+        }
+      } else {
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)